telosnetwork/telos-wallet
# Fixing blink of balances page when updating with indexer

# Fixes #538

## Description
This PR fixes the weird behavior that you can see on the balance page when the indexer is healthy (like right now on Mainnet).

## Test scenarios
- you need to run this branch locally with .evm pointing to Mainnet
`git fetch --all; git checkout 538-balances-blinks-when-updated-using-the-indexer; echo 'NETWORK="mainnet"' > .env; yarn; yarn dev`
- login with the Team Account
  - you should see a lot of tokens and their prices
  - TLOS should be on top
- wait for 20 sec
  - you should see no blink nor any weird effect
- now login using a brand new Account with no balances
  - you should see at least the three system tokens with cero balances

## Screenshots

### Team Account nalances

![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/ddd5ec95-8d45-4863-a52c-454ce7988468)

### Brand new Account balances

![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/48864e18-1cf9-42a1-919e-69a8b64452c3)

## Checklist:
